dashpay/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
     id 'kotlin'
 }
 
-//version '0.11-SNAPSHOT'
+version '0.14-SNAPSHOT'
```

Comment:
Is this var being used somewhere? It looks like we could use it below on lines 18, 19.

Reply:
I thought this version variable was used to define the version of the artifact that will be created by gradle.  Though, this commented out previously.